### PR TITLE
Update/gas estimation

### PR DESCRIPTION
**Pull request by Kayanski**
This PR aims at having a better gas estimation accounting for low gas transactions that need a higher buffer for passing

### CODE REVIEW

File: cw-orch/src/daemon/tx_builder.rs [hunk 1]
```diff
@@ -98,7 +100,11 @@ impl TxBuilder {
                 .await?;
             log::debug!("Simulated gas needed {:?}", sim_gas_used);
 
-            let gas_expected = sim_gas_used as f64 * GAS_BUFFER;
+            let gas_expected = if sim_gas_used < BUFFER_CHANGE_LIMIT {
+                sim_gas_used as f64 * SMALL_GAS_BUFFER
+            } else {
+                sim_gas_used as f64 * GAS_BUFFER
+            };
```
Comment (Buckram123) [inline cw-orch/src/daemon/tx_builder.rs:107]: Nice

**Review by Buckram123 — APPROVED** [review]
LGTM

File: cw-orch/src/daemon/tx_builder.rs [hunk 2]
```diff
@@ -10,6 +10,8 @@ use secp256k1::All;
 use super::{sender::Sender, DaemonError};
 
 const GAS_BUFFER: f64 = 1.2;
+const BUFFER_CHANGE_LIMIT: u64 = 100_000;
```
Comment (Buckram123) [inline cw-orch/src/daemon/tx_builder.rs:13]: :skull_and_crossbones: just got 
```
Error: tx failed: out of gas in location: wasm contract; gasWanted: 117885, gasUsed: 118671: out of gas with code 11
```
Should we increase it to 150/200k?

**Review by Buckram123 — CHANGES_REQUESTED** [review]

File: cw-orch/src/daemon/tx_builder.rs (hunk 2 above)
Comment (Kayanski) [reply cw-orch/src/daemon/tx_builder.rs:13]: Let's try 150k yeah

Comment (Kayanski) [reply cw-orch/src/daemon/tx_builder.rs]: Done,

**Review by Buckram123 — APPROVED** [review]
150 worked for me, hopefully we won't have 20 buffer ranges at the end

**Comment by CyberHoward** [comment]
bumped the threshold as discussed internally. Also bumped the base gas multiplier based on the chain-registry suggested gas multipliers. 

Most chains have 1.2 but some significant ones have 1.3. Since this is mainly a development and deployment tool I think reliability has much greater value than saving some cents here and there.

**Pull request merged**